include/experimental/__p2630_bits/strided_slice.hpp

```diff
@@ -15,19 +15,32 @@
 //
 //@HEADER
 
+#include <type_traits>
+
 namespace std {
 namespace experimental {
 
+namespace {
+  template<class T>
+  struct __mdspan_is_integral_constant: std::false_type {};
+
+  template<class T, T val>
+  struct __mdspan_is_integral_constant<integral_constant<T,val>>: std::true_type {};
+}
 // Slice Specifier allowing for strides and compile time extent
 template <class OffsetType, class ExtentType, class StrideType>
-struct strided_index_range {
+struct strided_slice {
   using offset_type = OffsetType;
   using extent_type = ExtentType;
   using stride_type = StrideType;
 
   OffsetType offset;
   ExtentType extent;
   StrideType stride;
+
+  static_assert(is_integral<OffsetType>::value || __mdspan_is_integral_constant<OffsetType>::value);
```

Comment:
Thanks for adding the Mandates! : - )

Optionally, given that the P2630 implementation only needs to support C++17, would you consider using `is_integral_v<T>` instead of `is_integral<T>::value`?

Reply:
I think I got confused with the error message for is_integra_constant_v which doesn't exist :-)

Reply:
changed that back.